### PR TITLE
Update non-root lanes to 1.21

### PR DESCRIPTION
**Pull request by xpivarc**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -979,50 +980,6 @@ presubmits:
...
-            - "/usr/local/bin/runner.sh"
-            - "/bin/sh"
-            - "-c"
-            - "automation/test.sh"
-          # docker-in-docker needs privileged mode
-          securityContext:
-            privileged: true
-          resources:
-            requests:
-              memory: "29Gi"
```
Comment (maiqueb) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:1025]: It's not obvious to me why this was deleted. 

Could you explain ?

Comment (fgimenez) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:1025]: rook-ceph lanes have been merged with sig-storage lanes, probably related

**Review by fgimenez — APPROVED** [review]
/approve
/hold

1.21 is not yet required, should we move the non-root lanes to it now or wait until it is?

**Review by dhiller — APPROVED** [review]

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 1 above)
Comment (xpivarc) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:1025]: @fgimenez yes, indeed.

**Comment by xpivarc** [comment]
> /approve
> /hold
> 
> 1.21 is not yet required, should we move the non-root lanes to it now or wait until it is?

Let`s just update to 1.20 for now. I had to test something with 1.19 but we can unhold now.

**Review by fgimenez — COMMENTED** [review]
/lgtm

**Review by fgimenez — COMMENTED** [review]
/unhold

**Pull request merged**